README.md

```diff
@@ -1,24 +1,69 @@
-# README
+# Bcycom Back End
+> This is the backend for the [Bcycom]() web application and below is what you can following to set up this web application...
```

Comment:
- [ ] The link is not working add the link https://github.com/Dagic-zewdu/final-capstone-frontend